src/pages/guides/continuous-developer-onboarding/index.module.scss

```diff
@@ -1,13 +0,0 @@
-@import '@styles/colors';
-
-.hero {
-    background-image: url('/guides/dev-onboarding/hero-mobile.png');
```

Comment:
This image & the one on line 10 can be removed from `/public` since we won't be referencing them anymore, right?

Reply:
Oh yes, great catch! They are removed in 11b7fd7. After removing the thumbnails, I also removed the unneeded directories in `public/guides` and updated the corresponding PDF paths.